### DISCUSSION TITLE
Unauthorized error while pull rabbitmq:3.9.13-management

### DISCUSSION THREAD

**Discussion by lanarchyste** [docker-library/rabbitmq#551] [Q&A] [answered]
The error is:
`Error response from daemon: Head "https://registry-1.docker.io/v2/library/rabbitmq/manifests/3.9.13-management": unauthorized: incorrect username or password`

**Comment by wglambert** [comment]
I'm not able to reproduce, could be an intermittent network/server issue or something with the host's environment
```console
$ docker pull rabbitmq:3.9.13-management
3.9.13-management: Pulling from library/rabbitmq
7c3b88808835: Already exists
1b19cf6b1093: Pull complete
9eb239857e4f: Pull complete
c50ef3b95025: Pull complete
d01396ec8261: Pull complete
8b7eaf109ebf: Pull complete
934bad7cb39a: Pull complete
10af7e8e268e: Pull complete
0deb0c230266: Pull complete
0ceb6133da69: Pull complete
Digest: sha256:17f46961d14264950fa71e79693cdd5c1d09c824fcc8a8aa4118c3c8f5cd1e9a
Status: Downloaded newer image for rabbitmq:3.9.13-management
docker.io/library/rabbitmq:3.9.13-management
```

**Comment by lanarchyste** [comment, marked as answer]
I fixed the problem by reconnecting with the following commands:
`docker logout`
`docker login --username [username]`

Thank you for taking the time to test on your side.
